README.md

```diff
@@ -1,9 +1,28 @@
+<div align="center">
+
+<img src="https://raw.githubusercontent.com/Sol-Client/client/develop/src/main/resources/assets/sol_client/textures/gui/icon.png" alt="Sol Client's logo">
+
 # Sol Client
-A nice Minecraft (non-hacked) client for 1.8.9 (and possibly newer versions in future - for an alternative [AxolotlClient](https://axolotlclient.github.io) is available)!
 
-<a href="https://discord.gg/TSAkhgXNbK"><img alt="Join Discord" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/cozy/social/discord-plural_vector.svg"/></a>
+An **open source**, non-hacked client for Minecraft 1.8.9 (newer versions coming in the future)
+
+<a href="https://www.java.com"><img alt="Java" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/cozy/built-with/java_vector.svg"></a>
+<a href="https://discord.gg/TSAkhgXNbK"><img alt="Join Discord" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/cozy/social/discord-plural_vector.svg"></a>
+<a href="https://sol-client.github.io"><img alt="Website" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/cozy/documentation/website_vector.svg"></a>
+
+</div>
+
+## 🤔 Why does this exist? I mean, Lunar seems to be good enough..
+
+Sol Client is one of the _very very_ few Minecraft 1.8 clients that is open source and privacy friendly. This means that the code written for client to be made is available to the public, thus it's safe. So, anyone could make sure that Sol is not a virus or it doesn't spy on you. Lunar on the hand is the exact opposite. It is propriety software and **it spies on you** and they even say it on their website:
+
+**From: https://www.lunarclient.com/privacy**
+> We may also **create or collect device-identifiable information** (DII), such as cookies, statistical identifiers, **unique device and advertising identifiers**, **usernames**, and similar identifiers that are linkable to a browser or device. From these platforms, we may also receive other information, such as **your IP address**, user agent, timestamps, **precise and imprecise geolocation**, **sensor data**, **apps**, fonts, battery life information and **screen size**.
+
+Not to mention the active developmenrt and that **every** suggested feature is considered and most likely added or planned to be added.
```

Comment:
I don't think Lunar's issues belong here to be honest. People can find information about it on other websites, and (it's surprising, but) I didn't know of Lunar's existence when I begun development. Still, I think a "Why" section stating that it exists because of its ease of use and consistency compared to many modpacks would be good (i.e. in modpacks you will often have many different configuration libraries)!

Reply:
I'll modify it then